assets/js/romo/dropdown.js

```diff
@@ -41,7 +41,9 @@ RomoDropdown.prototype.doPopupOpen = function() {
   }
 
   Romo.addClass(this.popupElem, 'romo-dropdown-open');
-  this.doPlacePopupElem();
+  setTimeout(Romo.proxy(function() {
```

Comment:
@jcredding we discussed not wrapping in the proxied anonymous function and just adding directly.